lib/ts/recipe/thirdpartyemailpassword/recipe.ts

```diff
@@ -95,6 +91,15 @@ export default class Recipe extends RecipeModule {
             this.apiImpl = builder.override(this.config.override.apis).build();
         }
 
+        /**
+         * emailDelivery will always needs to be declared after isInServerlessEnv
+         * and recipeInterfaceImpl values are set
+         */
+        this.emailDelivery =
+            ingredients.emailDelivery === undefined
+                ? new EmailDeliveryIngredient(this.config.getEmailDeliveryConfig())
```

Comment:
To this, you can give `EmailPasswordRecipeImplementation(this.recipeInterfaceImpl)` instead of `emailpasswordInsatnce.recipeImpl`.